docs/asyncapi.yaml

```diff
@@ -102,22 +102,33 @@ components:
       type: string
       description: The name of the source system that originated the message
       example: "AHWP"
+      minleght: 3
+      maxlenght: 100
+      pattern: "^[a-z0-9-_]+$"
 
     notifytemplateid:
       type: string
       description: Notification template identifier
       example: "f33517ff-2a88-4f6e-b855-c550268ce08a"
+      minleght: 3
+      maxlenght: 100
+      format: uuid
 
     commstype:
       type: string
       description: Type of communication (e.g., email, SMS)
       default: "email"
       example: "email"
+      minleght: 3
+      maxlenght: 50
 
     commsaddress:
       type: string
       description: The user's contact email address
       example: "mrMegaFarmer@strawberryFortFarm.co.uk"
+      minleght: 3
+      maxlenght: 100
+      pattern: "^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\\.[a-zA-Z]{2,}$"
```

Comment:
Does this match Notify's validation?

Reply:
I have not checked this. And couldn't find anything in their docs. This pattern should have followed a@a.co.uk pattern, but I can see that I dont handle second "." case. Do we have access to notify's validation logic?

Reply:
Can snoop through their code